lib/mpu6050/MPU6050.cpp

```diff
@@ -67,7 +67,8 @@ void MPU6050::initialize(uint8_t address) {
  */
 bool MPU6050::testConnection() {
     uint8_t deviceId = getDeviceID();
-    return deviceId == 0x68 || deviceId == 0x70 || deviceId == 0x71 || deviceId == 0x73; // Allow any MPUs
+    // some MPU-6500 seem to have their ID as 0x74
+    return deviceId == 0x68 || deviceId == 0x70 || deviceId == 0x71 || deviceId == 0x73 || deviceId == 0x74; // Allow any MPUs; 
```

Comment:
If we add any comment to 1 ID, i think we should name them all. Also we do no longer any MPUs
```suggestion
    // 0x68 -> MPU-6050
    // 0x70 -> MPU-6500
    // 0x71 -> MPU-9250
    // 0x73 -> MPU-9255
    // 0x74 -> MPU-6515
    return deviceId == 0x68 || deviceId == 0x70 || deviceId == 0x71 || deviceId == 0x73 || deviceId == 0x74; 
```